jupyterhub/values.yaml

```diff
@@ -355,6 +355,7 @@ prePuller:
     image:
       name: jupyterhub/k8s-image-awaiter
       tag: 'set-by-chartpress'
+    strictCheckDuration: 10
```

Comment:
My suggestion is to keep this at -1, which is the current default. Then we can run this with a 10s timeout in production for a couple weeks, and then change the default if that works. How does that sound?

Reply:
Hmmm yes we can do that, but i consider it a bug to fail as it currently is where a full node can make a deployment crash - due to that i believe it could make sense to go for the fix by default.

Although, this PR could introduce some other bug, so then its ansafeguard for not making it worse. Sure! Going for that